### ISSUE TITLE
Set preview attributes with an option

### ISSUE THREAD

**Issue by bohni** [asciidoctor/asciidoctor-vscode#155]
It would be great to set the URL of the plantuml server as a setting, not as variable in the asciidoc document itself.

**Comment by anoff** [comment]
Any ideas where/how to fix this? Would be happy to contribute, but have no experience with vscode plugins 😞

**Comment by MatteoCampinoti94** [comment]
We could add a settings option to add attributes to the preview. We could have an option like this:
```json
"asciidoc.preview.attributes": {
  "attribute1": "value1",
  "attribute2": "value2",
  "attribute3": "value3"
}
```

**Comment by MatteoCampinoti94** [comment]
It would require you to always add the value field, empty if necessary, but otherwise it would support unsetting and not-overriding (`attribute@`) attributes.

**Comment by MatteoCampinoti94** [comment]
Closed with release 2.6.0 #189
